lib/msf/core/db_manager/host.rb

```diff
@@ -73,14 +88,35 @@ def add_host_tag(opts)
   # ATM it will delete the tag from the tag table, not the host<->tag link
   def delete_host_tag(opts)
     wspace = Msf::Util::DBManager.process_opts_workspace(opts, framework)
-
-    host_id = opts[:id]
     tag_name = opts[:tag_name]
     tag_ids = []
 
-    host = wspace.hosts.find(host_id)
```

Comment:
Same logic as before if possible: https://github.com/rapid7/metasploit-framework/pull/17576/files#r1094675399

Reply:
This is in deleted code so not sure what the idea was here?

Reply:
Should be fixed in https://github.com/rapid7/metasploit-framework/pull/17576/commits/2df445152766e46b99c1494fce5e67d87d022586